dataRetriever/unitType.go

```diff
@@ -51,6 +51,8 @@ const (
 	ScheduledSCRsUnit UnitType = 22
 	// ProofsUnit is the header proofs unit identifier
 	ProofsUnit UnitType = 23
+	// ProofsNonceHashDataUnit is the proofs nonce-hash pair data unit identifier
+	ProofsNonceHashDataUnit UnitType = 24
```

Comment:
can you check if we can use the same header nonce to hash storer instead of a new one?

Reply:
updated to reuse them